roles/appsuite/templates/opt/open-xchange/etc/dbconnector.yaml.j2

```diff
@@ -0,0 +1,15 @@
+{{ ansible_managed | comment }}
+
+com.mysql.jdbc:
+    useUnicode: true
+    characterEncoding: UTF-8
+    autoReconnect: false
+    useServerPrepStmts: false
+    useTimezone: true
+    serverTimezone: UTC
+    connectTimeout: {{ appsuite_dbconnector_connect_timeout }}
+    socketTimeout: {{ appsuite_dbconnector_socket_timeout }}
+    useSSL: false
+    requireSSL: false
+    verifyServerCertificate: false
+    enabledTLSProtocols: TLSv1,TLSv1.1,TLSv1.2
```

Comment:
Could you parameterize those values and not hardcode them? Or will those values never be changed?

Reply:
So far we've only been parametrizing the values we actually needed to change, simply due to the sheer amount of config options that AppSuite has.
Chances are the other values in this file won't need changing anytime soon.

Reply:
They are now all templated with their former hardcoded values as defaults.